WikiWatershed/model-my-watershed
# Gather data from non-raster datasets

## Overview

Gathers data from non-raster datasets for populating a GWLF-E model.

 - [x] Weather Stations
 - [x] National Weather
 - [x] Streams
 - [x] Animal Types
 - [x] Point Source Discharge

## Testing Instructions

Check out the branch and open mmw. Create a new project with the MapShed model, and inspect the JSON result of the job. It should have a large blob filled with data from the defaults as well as the non-raster datasets.

## Code Review Instructions

We are attempting raw SQL queries for the first time in this project. Comment on the code style used there.

Using the GMS File Key spreadsheet in the MapShed folder on Google Drive, and the Class1.vb source file of the 1.3.0 version, do a quick spot check to ensure that the variables are calculated correctly.

## Output

Sample output from 
[Uploading MapShed.Output.json.txt…]()




## Notes

The raster fields will follow in an upcoming PR.

Connects #1227
